src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -686,6 +695,12 @@ export default class FilingHistoryList extends Vue {
         item.expiry = filing.data.restoration?.expiry
       }
 
+      // add properties for limited restoration extensions
+      if (EnumUtilities.isTypeRestorationExtension(filing)) {
+        item.isTypeLimitedRestorationExtension = true
+        item.expiry = filing.data.restoration?.expiry
+      }
```

Comment:
This should be OK since a limited restoration already returns `expiry` like this.